### PR TITLE
[#120455] Show error when trying to reactivate split with suspended child

### PR DESCRIPTION
**Pull request by jhanggi**
I tried to use a validation, but ended up not just overriding the `unsuspend` method because the ordering gets complicated between the validation and `:set_suspended_at_from_subaccounts`.

### CODE REVIEW

File: spec/models/account_spec.rb [hunk 1]
```diff
@@ -147,10 +147,10 @@
   it "should set suspend_at on suspend and unsuspend" do
     @owner   = FactoryGirl.create(:user)
     @account = FactoryGirl.create(:nufs_account, account_users_attributes: account_users_attributes_hash(user: @owner))
-    assert_nothing_raised { @account.suspend! }
-    expect(@account.suspended_at).not_to eq(nil)
-    assert_nothing_raised { @account.unsuspend! }
-    expect(@account.suspended_at).to eq(nil)
+    expect(@account.suspend).to be_truthy
```
Comment (benlinton) [inline spec/models/account_spec.rb]: ` expect(@account.suspend).to eq(true)` to be more explicit?

File: spec/models/account_spec.rb [hunk 2]
```diff
@@ -147,10 +147,10 @@
...
+    expect(@account.suspended_at).to be_present
+    expect(@account.unsuspend).to be_truthy
+    expect(@account.suspended_at).to be_blank
```
Comment (benlinton) [inline spec/models/account_spec.rb]: `be_nil`

File: spec/models/account_spec.rb [hunk 3]
```diff
@@ -147,10 +147,10 @@
   it "should set suspend_at on suspend and unsuspend" do
     @owner   = FactoryGirl.create(:user)
     @account = FactoryGirl.create(:nufs_account, account_users_attributes: account_users_attributes_hash(user: @owner))
-    assert_nothing_raised { @account.suspend! }
-    expect(@account.suspended_at).not_to eq(nil)
-    assert_nothing_raised { @account.unsuspend! }
-    expect(@account.suspended_at).to eq(nil)
+    expect(@account.suspend).to be_truthy
+    expect(@account.suspended_at).to be_present
+    expect(@account.unsuspend).to be_truthy
```
Comment (benlinton) [inline spec/models/account_spec.rb]: `eq(true)`

File: spec/models/account_spec.rb (hunk 1 above)
Comment (jhanggi) [inline spec/models/account_spec.rb]: I don't have a strong feeling either way. I feel like @drench has pushed me this way before.

**Comment by benlinton** [comment]
Minor comments.  Looks good :+1:

File: spec/models/account_spec.rb (hunk 1 above)
Comment (benlinton) [inline spec/models/account_spec.rb]: `eq(true)` is more likely to catch a bug if one pops up, but overall it's not a big deal.  :+1:

**Pull request closed without merging**